livre/eq/Node.cpp

```diff
@@ -129,6 +119,17 @@ class Node
         _textureDataCachePtr.reset();
     }
 
+    void updateAndSendFrameRange()
```

Comment:
according to some coding document the function name should always explain what is going in it, which I am generally trying to keep.